extensions/LoserFox/post-process.js

```diff
@@ -0,0 +1,753 @@
+(function(Scratch) {
```

Comment:
While using this extension, i've come upon a huge problem. When i use the effect, it literally freezes the screen and then when i toggle the effect away or there's some sort of movement, the project unfreezes itself. i'll link under here a video.

[video_3.webm](https://github.com/TurboWarp/extensions/assets/69966423/d87d2fde-5d9c-48af-8f71-30be3348117e)